subsys/usb/class/bluetooth.c

```diff
@@ -167,32 +169,97 @@ static void hci_rx_thread(void)
 	}
 }
 
+static uint16_t hci_pkt_get_len(struct net_buf *buf,
+				const uint8_t *data, size_t size)
+{
+	uint16_t len = 0;
+	size_t hdr_len = 0;
+
+	switch (bt_buf_get_type(buf)) {
+	case BT_BUF_CMD: {
```

Comment:
Are buffer types `BT_BUF_CMD` and `BT_BUF_ISO_OUT` here for future purposes? I don't see them used yet.

Reply:
I understood this now, `BT_BUF_CMD` and `BT_BUF_ISO_OUT` are needed for `BT_BUF_H4` type.